lib/ecto/adapters/mnesia.ex

```diff
@@ -150,34 +155,44 @@ defmodule Ecto.Adapters.Mnesia do
     end
   end
 
-  @doc """
-  Insert Ecto Schema struct to Mnesia database.
-
-  TODO:
-  - Process `opts`.
-  - Process `on_conflict`
-  - Process `returning`
-  """
-  def insert(_repo, %{autogenerate_id: {pk_field, _pk_type}, schema: schema, source: {_, table}}, params,
+  @doc false
+  # Insert Ecto Schema struct to Mnesia database.
+  # TODO: deal with `opts`: `on_conflict` and `returning`
```

Comment:
Found a TODO tag in a comment: # TODO: deal with `opts`: `on_conflict` and `returning`